### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by n4-t3**
- Made drop down menu more visible
Before:
dropbefore
After:
dropafter
- Made the text field larger
Before:
before
After:
after

### CODE REVIEW

File: src/components/basic/input/Text.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const TextInput = View<Props>(({ placeholder, value, onChange, disabled =
   )
 })
 
-const TextInputComponent = styled.input`
```
Comment (jhw123) [inline src/components/basic/input/Text.tsx:25]: I actually reverted this change because it makes the height of this component inconsistent with other UI components (e.g., button). Do you find severe usability issue with this?

File: src/components/basic/input/Select.tsx [hunk 2]
```diff
@@ -48,7 +49,7 @@ const Options = styled.select`
   outline: none;
   font-size: inherit;
   background-color: transparent;
-  color: inherit;
```
Comment (jhw123) [inline src/components/basic/input/Select.tsx:51]: The background of the dropdown on my end was black so this worked... But coloring to black will make the text hard to see on blue background.

File: src/components/basic/input/Text.tsx (hunk 1 above)
Comment (n4-t3) [reply src/components/basic/input/Text.tsx:25]: > I actually reverted this change because it makes the height of this component inconsistent with other UI components (e.g., button). Do you find severe usability issue with this?

I feel like some contents are hard to follow because the line doesn't wrap and users have to drag the mouse all the way around the edge highlighting elements for those cases which is cumbersome. However, since consistency is important we can leave this because of the work around.

**Comment by n4-t3** [comment]
I am closing this pull request as the UI updates do not seem sufficiently important or contribute to workflow improvement.
